protostar/cheatable_starknet/cheatcodes/cairo_cheatcode.py

```diff
@@ -1,12 +1,56 @@
-from abc import ABC
-from typing import TYPE_CHECKING
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Callable, Union
+
+from typing_extensions import Literal
+from starknet_py.cairo.felt import encode_shortstring
 
 from protostar.cairo import HintLocal
+from protostar.cheatable_starknet.cheaters.transaction_revert_exception import (
+    TransactionRevertException,
+)
 
 if TYPE_CHECKING:
     from protostar.cheatable_starknet.cheaters import CairoCheaters
 
 
+@dataclass(init=False)
+class CairoCheatcodeInvalidExecution:
+    ok = None
+    err_code: int
+
+    def __init__(self, err_code: int):
+        assert err_code > 0
```

Comment:
please add an assertion message so it's easier to debug

Reply:
[describe assertion](https://github.com/software-mansion/protostar/pull/1404/commits/3d72611785130d5bcd8382b71504473658dea1f9)